spot-setup/Apache Spot Schema.md

```diff
@@ -0,0 +1,43 @@
+# Apache Spot Schema 
+
+This document is to centralize a place where users can read information about Proxy, DNS and Netflow schema.
+
+## Proxy
+
+*Raw data*
```

Comment:
I'm wondering if it contains both raw data and calculated columns, should be called Raw Data?